jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/json/JsonCfnService.kt

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudformation.json
+
+import com.intellij.json.psi.JsonFile
+import com.intellij.json.psi.JsonObject
+import com.intellij.openapi.components.ServiceManager
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.util.PsiTreeUtil
+import software.aws.toolkits.jetbrains.services.cloudformation.CFN_FORMAT_VERSION
+import software.aws.toolkits.jetbrains.services.cloudformation.CfnParsedFile
+import software.aws.toolkits.jetbrains.services.cloudformation.SERVERLESS_TRANSFORM
+
+/**
+ * Acts a proxy that hides all the logic that requires JSON plugin to be enabled
```

Comment:
Can you rewrite this comment, I had to read it a few times to understand what it said

Reply:
Ya